tests/e2e/DropDownFilter.spec.ts

```diff
@@ -0,0 +1,13 @@
+import defaultFilter from "./pages/default.page";
```

Comment:
The tests are not complete!
A minimal test is not acceptable for new widgets. 
Test should cover the "Container" component.